packages/fluentui/react-builder/src/components/Designer.tsx

```diff
@@ -1,11 +1,15 @@
 import * as React from 'react';
 import { useImmerReducer, Reducer } from 'use-immer';
-import { Text, Button, Divider } from '@fluentui/react-northstar';
+import { Text, Button, Divider, Loader } from '@fluentui/react-northstar';
+import { FilesCodeIcon, AcceptIcon } from '@fluentui/react-icons-northstar';
 import { EventListener } from '@fluentui/react-component-event-listener';
 import { renderElementToJSX } from '@fluentui/docs-components';
 
 import { componentInfoContext } from '../componentInfo/componentInfoContext';
 import { ComponentInfo } from '../componentInfo/types';
+import ComponentControlsCodeSandbox, {
+  CodeSandboxState,
+} from '../../../docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/ComponentControlsCodeSandbox';
```

Comment:
We can't use such import. We can move this component to `@fluentui/docs-components` and then reuse it there